src/facade/reply_builder.cc

```diff
@@ -360,6 +360,17 @@ void RedisReplyBuilder::SendScoredArray(const std::vector<std::pair<std::string,
   }
 }
 
+void RedisReplyBuilder::SendNumSubArray(const std::vector<std::pair<std::string, long>>& arr) {
+  ReplyAggregator agg(this);
+
+  StartArray(arr.size() * 2);
+  for (const auto& p : arr) {
+    SendBulkString(p.first);
+    SendLong(p.second);
+  }
+  return;
```

Comment:
No need to `return` in the end of the function

Reply:
fixed: https://github.com/dragonflydb/dragonfly/pull/2282/commits/244eb2ae454ee15c40554f85082bc606a385cf8f